tests/test_parsig_api.py

```diff
@@ -102,3 +102,9 @@ def test_parse_multiple_instructions3(self):
         result = self.sig_parser.parse(sig)
         expected = [first_expected, second_expected, third_expected]
         self.assertEqual(result, expected)
+        
+    def test_autocorrect(self):
+        sig = "Take 1 talbet 3 tiems a day for 2 wekes"
```

Comment:
Add at least 5 more tests please :)

Reply:
There are 7 tests.
I think there might be a better way to test the autocorrect feature, but I will revisit the test issue in the future as the code passed all of the current tests.